docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,326 @@
+        shape to what Iris users and developers are used to. It is suspected
+        that optimal performance will need new chunking strategies, but at time
+        of writing (``Jan 2022``) experience is still limited.
+
+.. todo:
+        Revisit when we have more information.
+
+Spatial operations on mesh data are more complex
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Detail: :doc:`operations`
+
+Indexing a mesh data array cannot be used for:
+
+#. Region selection
+#. Neighbour identification
+
+This is because - unlike with a structured data array - relative position in
+a mesh's 1-dimensional data arrays has no relation to relative position in
+space. We must instead perform specialised operations using the information in
+the connectivities present, or by translating the mesh into a format designed
+for mesh analysis such as VTK.
+
+Such calculations can still be optimised to avoid them slowing workflows, but
+the important take-away here is that **adaptation is needed when working mesh
+data**.
+
+
+How Iris Represents This
+========================
+
+..
+    Include API links to the various classes
+
+    Include Cube/Mesh printout(s)
+
+.. seealso::
+
+        Remember this is a prose summary. Precise documentation is at:
+        :mod:`iris.experimental.ugrid`.
+
+.. note::
+
+        At time of writing (``Jan 2022``), neither 3D meshes nor 3D elements
+        (volumes) are supported.
+
+The Basics
+----------
+The Iris :class:`~iris.cube.Cube` has several new members:
+
+* | :attr:`~iris.cube.Cube.mesh`
+  | The :class:`iris.experimental.ugrid.Mesh` that describes the
+    :class:`~iris.cube.Cube`\'s horizontal geography.
+* | :attr:`~iris.cube.Cube.location`
+  | ``node``/``edge``/``face`` - the mesh element type with which this
+    :class:`~iris.cube.Cube`\'s :attr:`~iris.cube.Cube.data` is associated.
+* | :meth:`~iris.cube.Cube.mesh_dim`
+  | The :class:`~iris.cube.Cube`\'s **unstructured dimension** - the one that
+    indexes over the horizontal :attr:`~iris.cube.Cube.data` positions.
+
+These members will all be ``None`` for a :class:`~iris.cube.Cube` with no
+associated :class:`~iris.experimental.ugrid.Mesh`.
+
+This :class:`~iris.cube.Cube`\'s unstructured dimension has multiple attached
+:class:`iris.experimental.ugrid.MeshCoord`\s (one for each axis e.g.
+``x``/``y``), which can be used to infer the points and bounds of any index on
+the :class:`~iris.cube.Cube`\'s unstructured dimension.
+
+.. todo: Cube printout
+
+The Detail
+----------
+How UGRID information is stored
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. todo: Mesh printout?
+
+* | :class:`iris.experimental.ugrid.Mesh`
+  | Contains all information about the mesh.
+  | Includes:
+
+  * 1-3 collections of :class:`iris.coords.AuxCoord`\s:
+
+    * | **Required**: :attr:`~iris.experimental.ugrid.Mesh.node_coords`
+      | The nodes that are the basis for the mesh.
+    * | Optional: :attr:`~iris.experimental.ugrid.Mesh.edge_coords`,
```

Comment:
It may be worth a special note that we still ***require*** these at present (a specific limitation of the current Mesh implementation)

Reply:
a0b405c3b4432e6e172616620cb6a1563b083a57